backend/apps/webui/main.py

```diff
@@ -243,33 +246,23 @@ def get_pipe_id(form_data: dict) -> str:
     return pipe_id
 
 
-def get_function_params(function_module, form_data, user, extra_params={}):
```

Comment:
Using a `{}` or `[]` as a default parameter in Python can lead to problems, because the scope of that parameter is outside the function definition (so if you update it the updates would persist)